src/components/layout/Layout.js

```diff
@@ -1,44 +1,40 @@
-import React, { useState } from "react";
-import SideMenu from "./SideMenu/SideMenu";
-import MenuContext from "../../store/menu-context";
-import {
-  StyledContent,
-  StyledDrawer,
-  StyledDrawerPaper,
-  StyledRoot,
-} from "./LayoutStyled";
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { StyledContent } from './LayoutStyled';
+import Menu from './Menu/Menu';
 
 const Layout = ({ children }) => {
-  const [menuCollapsed, setMenuCollapsed] = useState(false);
+	const [open, setOpen] = useState(false);
 
-  const menuToggleHandler = () => {
-    setMenuCollapsed((value) => !value);
-  };
+	const handleDrawerOpen = () => {
+		setOpen(true);
+	};
 
-  return (
-    <StyledRoot>
-      <MenuContext.Provider
-        value={{
-          isMenuCollapsed: menuCollapsed,
-          onMenuToggle: menuToggleHandler,
-        }}
-      >
-        <StyledDrawer
-          variant="permanent"
-          open={true}
-          menucollapsed={menuCollapsed ? 1 : 0}
-        >
-          <StyledDrawerPaper menucollapsed={menuCollapsed ? 1 : 0}>
-            <SideMenu
-              onMenuToggle={menuToggleHandler}
-              isMenuCollapsed={menuCollapsed}
-            />
-          </StyledDrawerPaper>
-        </StyledDrawer>
-        <StyledContent container>{children}</StyledContent>
-      </MenuContext.Provider>
-    </StyledRoot>
-  );
+	const handleDrawerClose = () => {
+		setOpen(false);
+	};
+
+	let location = useLocation();
+
+	useEffect(() => {
+		handleDrawerClose();
+	}, [location]);
+

```

Comment:
I don't think we need this functionality n the desktop version as we have plantly of space and we can show the menu with text.